### ISSUE TITLE
Got error when trying to run go build

### ISSUE THREAD

**Issue by Fredy-Z** [kubernetes/sample-controller#49]
I followed the `README` and tried to run `go build -o sample-controller .`, then I got the error below:
```
# k8s.io/sample-controller/pkg/generated/clientset/versioned/typed/samplecontroller/v1alpha1
pkg/generated/clientset/versioned/typed/samplecontroller/v1alpha1/samplecontroller_client.go:73:45: scheme.Codecs.WithoutConversion undefined (type serializer.CodecFactory has no field or method WithoutConversion)
```

Any idea on how to solve it?

**Comment by Fredy-Z** [comment]
I run `go mod vendor` and the problem was solved.

**Comment by adinunzio84** [comment]
Same, though I fixed by adding `-u`:
`go get -u k8s.io/sample-controller`

**Comment by ju187** [comment]
I have a similar project and got this error after running generate-groups.sh to create clientset.  
go version 1.12.4
The project structure was initially setup with kubebuilder.  I noticed this line was updated recently 
config.NegotiatedSerializer = scheme.Codecs.WithoutConversion()
Where there be any issue if I revert to the previous code?
config.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: scheme.Codecs}
